packages/mobile/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
This component belongs to another functionality, please remove it from this branch